bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/api/Car.java

```diff
@@ -45,6 +46,7 @@ public class Car {
     private boolean disableHvac = false;
     private boolean disableLockStatus = false;
 
+    private OnOffType pausemode = OnOffType.OFF;
```

Comment:
It seems that this class in general abstracts openHAB types, so probably this should be:
```suggestion
    private boolean pausemode = false;
```